src/main/java/careconnect/logic/parser/ParserUtil.java

```diff
@@ -135,13 +138,26 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
      *
      * @throws ParseException if the given {@code date} is invalid.
      */
-    public static Date parseLogDate(String date) throws ParseException {
```

Comment:
not sure if we want to standardize singapore time everywhere.

Timezone should be "Asia/Singapore"